### DISCUSSION TITLE
MQTT on 0.1.4 - no motion detected

### DISCUSSION THREAD

**Discussion by B42S** [roleoroleo/yi-hack-Allwinner-v2#83] [Q&A]
Dear Roleoroleo,

first of all, thank you for the updated version. I'm really keen on the human detection and MQTT integration into my smarthome.
I was previously using 0.1.2 with motion detection and MQTT on Yi 1080p Home BFUS.
This was the one where human detection was always reported as "detected" on MQTT.

I did the installation of 0.1.4 from scratch on one of my Yi 1080p Home BFUS.

With 0.1.4 I'm experiencing that motion and humans are always reported as "not detected" and sound detection is reported once as detected after rebooting the camera.
Has anybody else the same issue?

Please find 2 screenshots attached:
-) MQTT_config.jpg  .... Topics config of MQTT (same config for all Cams)
-) MQTT-Explorer.jpg ... MQTT Explorer Screenshot (Cam2 still on 0.1.2 and Cam1 on 0.1.4)

Happy to contribute with more details and hoping that the problem is only on my side.

Thank you,
Bernhard

![MQTT_config](https://user-images.githubusercontent.com/78260498/106356193-0761d780-62fe-11eb-9dd2-778685a7916d.jpg)
![MQTT-Explorer](https://user-images.githubusercontent.com/78260498/106356194-09c43180-62fe-11eb-9a0b-2b3d052e2d00.jpg)

**Comment by roleoroleo** [comment]
There are a lot of posts about this issue.
Sorry but I think this feature is not working, both with the original firmware and with the hack.
https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/75
